#ifndef __PLUGIN_REGISTER_HANDLER_H__
#define __PLUGIN_REGISTER_HANDLER_H__
#define private public
#define protected public

#include "gtest/gtest.h"
class PluginRegisterHandlerTest : public testing::Test {
public:
    void SetUp()
    {
    }

    void TearDown()
    {
    }

    static void SetUpTestCase()
    {
    }

    static void TearDownTestCase()
    {
    }
};

#endif